themes/theme-gmd/components/View/components/Content/index.jsx

```diff
@@ -38,14 +38,9 @@ class ViewContent extends Component {
       keyboardHeight: 0,
     };
 
-    event.addCallback(EVENT_KEYBOARD_WILL_CHANGE, this.handleKeyboardChange);
-  }
-
-  /**
-   * Updates the content reference within the view provider.
-   */
-  componentDidMount() {
     this.props.setContentRef(this.ref);
+
+    event.addCallback(EVENT_KEYBOARD_WILL_CHANGE, this.handleKeyboardChange);
```

Comment:
This `Content` component now differs from the one from iOS. Here the ref is registered within the constructor, on iOS it happens within `componentDidMount`

Reply:
Normalised